soc/arm/raspberrypi/rp2/Kconfig.soc

```diff
@@ -0,0 +1,26 @@
+# Raspberry Pi RP2XXX MCU line
+
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2021 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "RP2xxx MCU Selection"
+	depends on SOC_SERIES_RP2XXX
+
+config SOC_RP2040
+	bool "Raspberry Pi RP2040"
+
+endchoice
+
+config RP2_FLASH_W25Q080
```

Comment:
Where are these used?

Reply:
In the hal's cmake, where the correct assembly file for the used flash should be chosen for the second stage bootloader